plugins/filter/crc32.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Julien Riou <julien@riou.xyz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleFilterError
+from ansible.module_utils.six import string_types
+from ansible.module_utils.common.text.converters import to_bytes
+
+try:
+    from zlib import crc32
+    HAS_ZLIB = True
+except ImportError:
+    HAS_ZLIB = False
+
+
+def crc32s(value):
+    if not isinstance(value, string_types):
+        raise AnsibleFilterError('Invalid value type (%s) for crc32 (%s)' %
```

Comment:
Better use
```suggestion
        raise AnsibleFilterError('Invalid value type (%s) for crc32 (%r)' %
```
so the representation of `value` is inserted (instead of its value).

Reply:
Type changed